### PR TITLE
Fix Code Quality Issues

### PR DESCRIPTION
**Pull request by andrea-pratt**

### CODE REVIEW

**Comment by claraj** [comment]
@omarmohamud23 can you review?

**Comment by omarmohamud23** [comment]
@apratt101300 Spacing and everything looks good, would suggest keeping your quotes consistent. Either single or double.

**Comment by andrea-pratt** [comment]
@omarmohamud23 Thanks for the suggestions. I went in and made sure all quotes were consistent. I believe there were only 2 or 3 lines that I changed if you'd like to review.

**Comment by andrea-pratt** [comment]
@claraj Could you please review my pull request for any necessary changes? Thank you!

File: convert_state.py [hunk 1]
```diff
@@ -1,60 +1,111 @@
...
+            print("Invalid entry.")
+  
+
+def convert_state_to_abvr(states):
+    """
+    Takes a state name and converts it to a state code
+
+    """
+    state = input("Enter state name: ").capitalize()
+    abrv_name = states.get(state)
```
Comment (claraj) [inline convert_state.py]: abbreviation or abbreviation_name would be a better name for this variable, same with this function name, also modify other names with abrv in.  If I have to remember this variable name, is it abbr_name or abrv_name or ab_name or ? And when reading, "abrv" may not be obvious that is is an abbreviation of "abbreviation". If the variable is called abbreviation_name it's easier to remember and it's clearer what it represents.

File: convert_state.py [hunk 2]
```diff
@@ -1,60 +1,111 @@
...
+    Reads a json file a returns a python dictionary
 
+    """
+    with open(states_file,"r") as file:
+        states = file.read()
+        abrvs_to_states = json.loads(states) 
+    return abrvs_to_states
 
 
+def reverse_dict(initial_dict):
```
Comment (claraj) [inline convert_state.py:81]: These utility functions are a good idea

File: convert_state.py [hunk 3]
```diff
@@ -1,60 +1,111 @@
...
+def show_menu_get_choice():
+    """
+    Displays the menu options and returns the user's choice
+
+    """
+    print("1. Convert state to abbreviation")
+    print("2. Convery abbreviation to state")
+    print("3. Quit")
+
+    choice =   input("Enter the integer value of your choice: ")
```
Comment (claraj) [inline convert_state.py]: Too many spaces after the =

File: convert_state.py [hunk 4]
```diff
@@ -1,60 +1,111 @@
...
+
+    return choice
+
+
+        
 if __name__ == "__main__":
```
Comment (claraj) [inline convert_state.py]: Remove blank line

**Review by claraj — CHANGES_REQUESTED** [review]
This looks MUCH better!  Use of functions, and code is much cleaner and more readable. Avoid shortening variable names unless it's a very common and very clearly understood, so abbreviation_name or convert_state_to_abbreviation.

File: convert_state.py (hunk 4 above)
Comment (andrea-pratt) [reply convert_state.py]: Sounds good.

File: convert_state.py (hunk 3 above)
Comment (andrea-pratt) [reply convert_state.py]: Yes. I didn't notice that one. Thanks!

File: convert_state.py (hunk 1 above)
Comment (andrea-pratt) [reply convert_state.py]: That makes sense.

**Comment by andrea-pratt** [comment]
@claraj I believe I've made all suggested changes. Let me know if there's anything else. Thanks!

**Pull request closed without merging**